### PR TITLE
Notifications: Disable notifications polling when the feature is disabled

### PR DESCRIPTION
**Pull request by nick-funk**
## What does this PR do?

- disables the `useLiveNotificationsPolling` hook when `inPageNotifications` is not enabled in the tenant settings

## These changes will impact:

- [X] commenters
- [ ] moderators
- [ ] admins
- [ ] developers

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

N/A

## How do I test this PR?

- enable in page notifications in the admin
- visit stream while logged in, see that polling is happening in the network tab of dev tools
- disable in page notifications in the admin
- visit stream while logged in, see that polling is not happening in the network tab of dev tools

## Were any tests migrated to React Testing Library?

No

## How do we deploy this PR?

Merge into epic branch.

### CODE REVIEW

File: client/src/core/client/stream/tabs/Notifications/polling/useLiveNotificationsPolling.tsx [hunk 1]
```diff
@@ -43,7 +58,7 @@ export default function useLiveNotificationsPolling(userID?: string) {
     `);
 
   const updater = useCallback(async () => {
-    if (!userID) {
+    if (!userID || !settings?.inPageNotifications?.enabled) {
```
Comment (kabeaty) [inline client/src/core/client/stream/tabs/Notifications/polling/useLiveNotificationsPolling.tsx]: Nice! Should we also check that in-page notifications is enabled in the viewer's preferences? (`viewer?.inPageNotifications?.enabled`)

Comment (nick-funk) [reply client/src/core/client/stream/tabs/Notifications/polling/useLiveNotificationsPolling.tsx]: oooh! good catch! I will update

Comment (nick-funk) [reply client/src/core/client/stream/tabs/Notifications/polling/useLiveNotificationsPolling.tsx]: fixed!

**Review by kabeaty — APPROVED** [review]
Looks good, thanks for the update!

**Pull request merged**